packages/pn-personafisica-webapp/src/component/Contacts/SpecialContacts.tsx

```diff
@@ -57,9 +73,10 @@ const SpecialContacts = ({ recipientId, legalAddresses, courtesyAddresses }: Pro
   const { initValidation } = useDigitalContactsCodeVerificationContext();
   const parties = useAppSelector((state: RootState) => state.contactsState.parties);
   const isMobile = useIsMobile();
+  const [inputValue, setInputValue] = useState('');
```

Comment:
Make the names more specific. It's too generic. Example: senderInputValue and setSenderInputValue

Reply:
do it